serverless-functions/requirements.txt

```diff
@@ -0,0 +1,9 @@
+azure-functions
+azure-identity
+azure-storage-blob
+hl7
+phdi
```

Comment:
We may need to change this to point to the main branch of PHDI while we are still working on the containers, as the PyPi release won't have that yet.

Reply:
This may not have to happen now, as this function isn't tied to the containers, but eventually as we integrate the other BB containers into the Azure project.

Reply:
Yeah I think we are okay using PyPi here since we are just grabbing one existing function from the SDK.

Reply:
However, thanks for reminding me that I can clean up some of these requirements that aren't needed!